kdwsdl2cpp/src/main.cpp

```diff
@@ -67,6 +67,12 @@ static void showHelp(const char *appName)
             "                            use of the import-path option\n"
             "  -help-on-missing          When groups or basic types could not be found, display\n"
             "                            available types (helps with wrong namespaces)\n"
+#if QT_VERSION >= QT_VERSION_CHECK(5, 4, 0)
+            "  -pkcs12file               Load a certificate from a PKCS12 file. You can use this option\n"
+            "                            if the WSDL file (or files refering to it) is served from a \n"
+            "                            location which require certificate based authentication\n"
+            "  -pkcs12password           Pass the password for the certificate file if required\n"
```

Comment:
Please add a note that this option is not secure and should be used with caution. Any user on the same machine can use `ps` to see that password as a command-line argument of this process.

I can't think of a better solution though, without adding a dependency on Qt5KeyChain or equivalent

Well, reading the password from a file would already be more secure, since you could limit the permissions for that file to the current user.

Reply:
> I can't think of a better solution though, without adding a dependency on Qt5KeyChain or equivalent

We could ask the user for type in the password, if the opening of the cert without password fails.

Reply:
Having to type the password at every build sounds rather annoying ;)

Reply:
Yes indeed. Let's keep it simple by adding a warning to the help.